Tests/SonarQube.Bootstrapper.Tests/BootstrapperExeTests.cs

```diff
@@ -154,7 +154,10 @@ public void Exe_PreProc_VersionCheckSucceeds_PreProcFails()
                 logger.AssertWarningsLogged(0);
 
                 string logPath = DummyExeHelper.AssertDummyPreProcLogExists(binDir, this.TestContext);
-                DummyExeHelper.AssertExpectedLogContents(logPath, "/install:true\r\n/d:sonar.host.url=http://host:9\r\n/d:another.key=will be ignored\r\n");
+                DummyExeHelper.AssertExpectedLogContents(logPath,
+                    "/install:true",
+                    "/d:sonar.host.url=http://host:9",
+                    "/d:another.key=will be ignored");
```

Comment:
I also changed "AssertExpectLogContents" to make it easier see what the expected arguments are.